generateNotesKendoAngular.js

```diff
@@ -0,0 +1,26 @@
+const packageJson = require('../../../package.json');
+const generateNotes = require('./generateNotes');
+
+module.exports = function (pluginConfig, {pkg}, cb) {
+
+    const themeVersionSectionDecorator = function() {
+    const devDeps = packageJson.devDependencies;
+    let log = arguments[1];
+    let themesSection = "";
+
+    for (const pack in devDeps) {
+        if (pack.match(/@progress\/kendo-theme-/)) {
+            themesSection += `* ${pack}: ${devDeps[pack]}\n`;
+        }
+    }
+
+    if (themesSection.length) {
+        themesSection = "#### Themes supported\n\n" + themesSection;
```

Comment:
"Themes supported" sounds strange. I would suggest to go for "Supported themes" or "Supported theme versions".

Reply:
... [moving here](https://github.com/telerik/kendo-build-tasks/pull/227) .